tgstation.dme

```diff
@@ -1352,6 +1352,7 @@
 #include "code\modules\mob\living\simple_animal\shade.dm"
 #include "code\modules\mob\living\simple_animal\simple_animal.dm"
 #include "code\modules\mob\living\simple_animal\spawner.dm"
+#include "code\modules\mob\living\simple_animal\status_procs.dm"

```

Comment:
It could be worse, it just make simple_animal not immune to eye damage, ear damage, etc...
The only code in there is overriden procs just doing "return".